src/openrct2/ParkImporter.h

```diff
@@ -64,8 +64,9 @@ namespace ParkImporter
     [[nodiscard]] std::unique_ptr<IParkImporter> CreateS6(IObjectRepository& objectRepository);
     [[nodiscard]] std::unique_ptr<IParkImporter> CreateParkFile(IObjectRepository& objectRepository);
 
-    bool ExtensionIsRCT1(const std::string& extension);
-    bool ExtensionIsScenario(const std::string& extension);
+    bool ExtensionIsOpenRCT2ParkFile(std::string_view extension);
```

Comment:
Note: I chose to mention OpenRCT2, since `ExtensionIsParkFile` could be confused with "this function returns true for parks that are not scenarios".

Reply:
I don’t think that will be likely. We tend to call non-scenarios "saves", not "parks".

Reply:
I disagree and this that it is likely, especially since the function definition is right there with `ExtensionIsScenario`, so a more distinctive descriptor is desirable here.

Reply:
Can’t say I really see the point, but I already have 8243721 other hills to die on.